
        
<div> 
			<h2 style =" margin-top: 5px;" >&emsp;&emsp;&emsp;&emsp;Name: Frances Aneth Rosales <br> &emsp;&emsp;&emsp;&emsp;Student Number: 2021044631 <br> &emsp;&emsp;&emsp;&emsp;SA2  </h2>
		
</div>
		<br><br><br>
        

# Rat Exploration DATA



# READ FILE

<section id="sectionmeandiff"> </section>

In [7]:
import pandas as pd

RatData = 'RatExploration_csvFile.csv'

df = pd.read_csv(RatData)
print(df)


    ID  Stimuli  Time  Unnamed: 3
0    1    Shape  2.00         NaN
1    2    Shape  0.75         NaN
2    3    Shape  1.25         NaN
3    4    Shape  1.00         NaN
4    5    Shape  1.50         NaN
5    6    Shape  1.25         NaN
6    7    Shape  1.75         NaN
7    8    Shape  0.50         NaN
8    9  Pattern  2.50         NaN
9   10  Pattern  3.25         NaN
10  11  Pattern  1.85         NaN
11  12  Pattern  3.05         NaN
12  13  Pattern  2.50         NaN
13  14  Pattern  3.00         NaN
14  15  Pattern  4.10         NaN
15  16  Pattern  3.75         NaN
16  17  Picture  4.25         NaN
17  18  Picture  4.10         NaN
18  19  Picture  4.25         NaN
19  20  Picture  4.40         NaN
20  21  Picture  3.75         NaN
21  22  Picture  3.05         NaN
22  23  Picture  3.20         NaN
23  24  Picture  5.00         NaN
24  25    Shape  2.00         NaN
25  26    Shape  0.75         NaN
26  27    Shape  1.25         NaN
27  28    Shape  1.00         NaN
28  29  Patter

<html>
	<head>
		<meta http-equiv='Content-Type' content='text/html; charset=utf-8' />
		<title>JASP</title>		<style>			p {margin-top:1em; margin-bottom:1em;}		</style>	</head>
	<body style='display: block; padding: 0px; margin: 0px; '>
<div style='padding: 0px 12px 12px; text-align: start; margin-bottom: 8.4px; margin-top: 0px; margin-left: 8.4px; margin-right: 8.4px; display: block; float: none; position: relative; '>
<div style="display:inline-block; ">
<div  ><h2 >ANOVA</h2></div><div style='padding: 0px; text-align: start; margin-bottom: 0px; margin-top: 0px; margin-left: 0px; margin-right: 0px; display: block; float: none; position: static; '>
<div style="display:inline-block; ">
<div  ></div><div style='padding: 0px 7.2px 0px 0px; text-align: start; margin-bottom: 7.2px; margin-top: 1.2px; margin-left: 0px; margin-right: 7.2px; display: block; float: none; position: static; '>
<div style="display:inline-block; ">
<div class="jasp-table-primitive jasp-display-primitive" ><table class="jasp-no-select" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(128, 128, 128); border-style: none; display: table; float: none; '><thead><tr><th colspan="12" class="" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 1px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none none solid; display: table-cell; float: none; text-align: left; '><div class="toolbar" ><div class="jasp-toolbar span-toolbar" ><span class="in-toolbar toolbar-clickable" >ANOVA - Time</span>
<div class="toolbar-button toolbar-clickable jasp-menu jasp-hide" ></div>
<div class="status complete" ></div>
</div>
</div>
</th>
</tr>
<tr><th colspan="2" class="string" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 1px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none none solid; display: table-cell; float: none; text-align: center; '>Cases</th>
<th colspan="2" class="number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 1px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none none solid; display: table-cell; float: none; text-align: center; '>Sum of Squares</th>
<th colspan="2" class="integer" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 1px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none none solid; display: table-cell; float: none; text-align: center; '>df</th>
<th colspan="2" class="number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 1px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none none solid; display: table-cell; float: none; text-align: center; '>Mean Square</th>
<th colspan="2" class="number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 1px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none none solid; display: table-cell; float: none; text-align: center; '>F</th>
<th colspan="2" class="pvalue" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 1px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none none solid; display: table-cell; float: none; text-align: center; '>p</th>
</tr>
</thead>
<tbody><tr><td class="value text new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '>Stimuli</td>
<td class="symbol text new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>44.527</td>
<td class="symbol number new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>2</td>
<td class="symbol number new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>22.263</td>
<td class="symbol number new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>62.089</td>
<td class="symbol number new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value p-value new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>&lt; .001</td>
<td class="symbol p-value new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
</tr>
<tr><td class="value text new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '>Residuals</td>
<td class="symbol text new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>11.833</td>
<td class="symbol number new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>33</td>
<td class="symbol number new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>0.359</td>
<td class="symbol number new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '></td>
<td class="symbol number new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value undefined new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '> </td>
<td class="symbol undefined new-group-row last-group-row" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
</tr>
<tr><td colspan="12" class="" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 2px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none none solid; display: table-cell; float: none; text-align: right; '></td>
</tr>
</tbody>
<tfoot><tr><td colspan="12" class="" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '><em>Note.</em>
 Type III Sum of Squares</td>
</tr>
</tfoot>
</table>
</div>
</div></div></div></div></div></div>	</body>
</html>

In [10]:
import pandas as pd
from statsmodels.formula.api import ols
import statsmodels.api as sm
from tabulate import tabulate

id_vars = ['Stimuli']
value_vars = ['Time']

data_analyzed = pd.melt(df, id_vars=id_vars, value_vars=value_vars, var_name='Condition', value_name='Seconds')

formula = 'Seconds ~ Stimuli'
analyzed_frame = ols(formula, data=data_analyzed).fit()
anova_df = sm.stats.anova_lm(analyzed_frame, typ=2)

print(tabulate(anova_df, headers='keys', tablefmt='pretty'))


+----------+--------------------+------+-------------------+-----------------------+
|          |       sum_sq       |  df  |         F         |        PR(>F)         |
+----------+--------------------+------+-------------------+-----------------------+
| Stimuli  | 44.526805555555555 | 2.0  | 62.08885876263249 | 6.531739771892149e-12 |
| Residual | 11.832916666666666 | 33.0 |        nan        |          nan          |
+----------+--------------------+------+-------------------+-----------------------+


# Interpretation:

 
<h5>

<li> Stimuli Group <br>
Sum of Squares (SS): 44.527: This illustrates the overall variation in the amount of time that each stimulus group spent exploring the chamber.
<br>
Degrees of Freedom (df): 2: There are two degrees of freedom associated with the stimuli group, indicating the number of independent comparisons being made.<br>

Mean Square (MS): 22.263: This represents the average variation in the amount of time that each stimulus group spent exploring the chamber.
<br>

<br><br>sss

<li> Residuals <br>
Sum of Squares (SS): 11.833: This represents the residual variability in the data after accounting the differences between stimuli.<br>

Degrees of Freedom (df): 33: The residuals show 33 degrees of freedom, which represents the variability that the stimuli group was unable to account for.<br>

Mean Square (MS): 0.359: This is the average unexplained variability in the time spent exploring the chamber.<br>



Since F (62.089) with greater variation between group means is indicated by a bigger F-ratio, is a metric that represents the ratio of variance within groups to variance between groups.
<br><br><br><br>
<b>Additional, pairs of comparisons could be carried out in light of the statistically significant results to determine which particular stimuli groups differ from one another significantly.

<html>
	<head>
		<meta http-equiv='Content-Type' content='text/html; charset=utf-8' />
		<title>JASP</title>		<style>			p {margin-top:1em; margin-bottom:1em;}		</style>	</head>
	<body style='display: block; padding: 0px; margin: 0px; '>
<div style='padding: 0px 7.2px; text-align: start; margin-bottom: 7.2px; margin-top: 7.2px; margin-left: 0px; margin-right: 7.2px; display: block; float: none; position: static; '>
<div style="display:inline-block; ">
<div  ><h3 >Descriptives</h3></div><div style='padding: 0px 7.2px 0px 0px; text-align: start; margin-bottom: 7.2px; margin-top: 1.2px; margin-left: 0px; margin-right: 7.2px; display: block; float: none; position: static; '>
<div style="display:inline-block; ">
<div class="jasp-table-primitive jasp-display-primitive" ><table class="jasp-no-select" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(128, 128, 128); border-style: none; display: table; float: none; '><thead><tr><th colspan="12" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 1px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none none solid; display: table-cell; float: none; text-align: left; '><div class="toolbar" ><div class="jasp-toolbar span-toolbar" ><span class="in-toolbar toolbar-clickable" >Descriptives - Time</span>
<div class="toolbar-button toolbar-clickable jasp-menu jasp-hide" ></div>
<div class="status complete" ></div>
</div>
</div>
</th>
</tr>
<tr><th colspan="2" class="string" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 1px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none none solid; display: table-cell; float: none; text-align: center; '>Stimuli</th>
<th colspan="2" class="integer" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 1px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none none solid; display: table-cell; float: none; text-align: center; '>N</th>
<th colspan="2" class="number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 1px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none none solid; display: table-cell; float: none; text-align: center; '>Mean</th>
<th colspan="2" class="number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 1px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none none solid; display: table-cell; float: none; text-align: center; '>SD</th>
<th colspan="2" class="number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 1px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none none solid; display: table-cell; float: none; text-align: center; '>SE</th>
<th colspan="2" class="number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 1px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none none solid; display: table-cell; float: none; text-align: center; '>Coefficient of variation</th>
</tr>
</thead>
<tbody><tr><td class="value text" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '>Pattern</td>
<td class="symbol text" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>12</td>
<td class="symbol number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>2.888</td>
<td class="symbol number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>0.684</td>
<td class="symbol number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>0.197</td>
<td class="symbol number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>0.237</td>
<td class="symbol number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
</tr>
<tr><td class="value text" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '>Picture</td>
<td class="symbol text" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>12</td>
<td class="symbol number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>3.954</td>
<td class="symbol number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>0.608</td>
<td class="symbol number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>0.175</td>
<td class="symbol number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>0.154</td>
<td class="symbol number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
</tr>
<tr><td class="value text" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '>Shape</td>
<td class="symbol text" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>12</td>
<td class="symbol number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>1.250</td>
<td class="symbol number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>0.489</td>
<td class="symbol number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>0.141</td>
<td class="symbol number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
<td class="value number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: right; '>0.391</td>
<td class="symbol number" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 0px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none; display: table-cell; float: none; text-align: left; '></td>
</tr>
<tr><td colspan="12" style='border-collapse: collapse; border-top-width: 0px; border-bottom-width: 2px; border-left-width: 0px; border-right-width: 0px; border-color: rgb(0, 0, 0); border-style: none none solid; display: table-cell; float: none; text-align: right; '></td>
</tr>
</tbody>
<tfoot></tfoot>
</table>
</div>
</div></div></div></div>	</body>
</html>

# Conclusion


<h5>

The analysis of the stimulus groups in our data yields informative conclusions. The Stimuli Group, which has a 2 degree of freedom (df) indicating the number of independent comparisons, and a Sum of Squares (SS) of 44.527, explains the overall variability in the time spent exploring the chamber among the various stimuli groups. The average variability of the exploration time across the stimuli groups is represented by the Mean Square (MS) of 22.263. Conversely, residuals, which have an average unexplained variability of 0.359 and an SD of 11.833 and 33 df, represent the unexplained or residual variability after accounting for variations across stimulus groups. The F-ratio of 62.089, which indicates the ratio of variance within and between groups, highlights notable variations in group averages. <br>

It's interesting to note that our data reveals that Pattern, Picture, and Shape all have normal distributions, but Shape deviates from the norm and suggests inconsistent behavior. These complex findings shed light on the dynamics of stimulus-induced exploration and emphasize the significance of taking particular stimuli characteristics into account when interpreting the results. Additionally, the Standard Deviation of Shape being the smallest implies less variability and more homogeneity in the time spent exploring for "Shape."
